src/controllers/v1/reports.js

```diff
@@ -23,4 +24,34 @@ module.exports = class Reports {
 			return error
 		}
 	}
+
+	/**
+	 * Get Report config
+	 * @method
+	 * @name reportFilterList
+	 * @param {Object} req - request data.
+	 * @param {String} req.query.decodedToken - user token.
+	 * @returns {JSON} - filter list.
+	 */
+
+	async reportData(req) {
+		try {
+			let entity_type = {}
+			const reportData = await reportService.getReportData(
+				req.decodedToken.id,
+				req.query.report_code,
+				req.query.report_role ? req.query.report_role : common.MENTEE_ROLE,
+				req.query.start_date ? req.query.start_date : '',
+				req.query.end_date ? req.query.end_date : '',
+				req.query.session_type ? req.query.session_type : common.ALL,
+				req.query.entity_type,
+				req.query.sort_column ? req.query.sort_column : '',
+				req.query.sort_type ? req.query.sort_type : '',
+				req.query.download_csv ? req.query.download_csv : false
+			)
```

Comment:
@sumanvpacewisdom pagination is not require ?

Reply:
@rakeshSgr we need , that includes in 3rd milestone , i will include it